flytekit/models/sagemaker/hpo_job.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import absolute_import
+
+from flyteidl.plugins.sagemaker import hyperparameter_tuning_job_pb2 as _pb2_hpo_job
+from flytekit.models import common as _common
+from flytekit.sdk.sagemaker import types as _sdk_sagemaker_types
```

Comment:
let's remove this.  model files are supposed to be 1:1 to their protobuf generated counterparts.  there should be no need to import anything else from flytekit.  take a look at some of the other ones to see how enums are handled for instance.

Reply:
SparkTask did the same tho (I actually referenced this when I wrote mine) https://github.com/lyft/flytekit/blob/a106ba7c3946837039b807d14f62fd2cc7d7c53b/flytekit/models/task.py#L616

Reply:
Also, when users write code they need to have these enums available in sdk. I think having this is reasonable

Reply:
@wild-endeavor 
I'll move it to models, is the expectation for users to access the models wrapper? I feel that's a bit awkward, what do you think?